src/nvim/option.c

```diff
@@ -598,23 +598,20 @@ void set_init_1(void)
     }
   }
 
-  /*
-   * 'maxmemtot' and 'maxmem' may have to be adjusted for available memory
-   */
+  // 'maxmemtot' and 'maxmem' are ignored in Neovim, but we set them to the
+  // amount of available memory.
   opt_idx = findoption((char_u *)"maxmemtot");
   if (opt_idx >= 0) {
```

Comment:
Thanks! Forget to remove this since I was planning to not completely remove the variables at first.